### PR TITLE
Add YunoHost icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/10920

**Popularity metric:**

The Similarwebrank is 269,570. See https://www.similarweb.com/website/yunohost.org

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Downloaded icon from https://techicons.dev/icons/yunohost , resized and combined path and rect.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -17451,6 +17451,11 @@
             "hex": "84BD00",
             "source": "https://www.yubico.com/wp-content/themes/coronado/img/icon.svg"
         },
+        {
+            "title": "YunoHost",
+            "hex": "000000",
+            "source": "https://icon.icepanel.io/Technology/svg/Yuno-Host.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:17457]: Is IcePanel an official source for YunoHost? I can't see anything on the homepage to suggest it is.
Looking at Yunohost's own documentation there _is_ an SVG - but it's a horizontal one.

Comment (CodeShakingSheep) [reply _data/simple-icons.json:17457]: Nah, IcePanel is not really an official source for YunoHost. I just thought I could save some work here as on their official website, they just have a png, see https://yunohost.org. As I found the svg on IcePanel which looked fine I didn't want to manually vectorize the png icon from their website.

So, what would be the correct procedure here? Manually vectorize the png from the website or just change the source? Personally, I prefer the compact logo over the horizontal logo from their documentation.

Comment (adamrusted) [reply _data/simple-icons.json:17457]: Ideally, we'd find an official source (one of their websites, a Wikipedia entry, or a GitHub repo of theirs) and use that as a source. Failing that, we can compare the SVG you've found against the PNG - but the source should _really_ come from the company themselves.

**Comment by adamrusted** [comment]
@CodeShakingSheep - please don't feel the need to merge develop into this unless you're making changes locally. Anyone that has left a comment or review gets pinged every time you do that. 😄

File: _data/simple-icons.json (hunk 1 above)
Comment (Tagadda) [reply _data/simple-icons.json:17571]: :eyes: https://github.com/YunoHost/doc/blob/master/images/ynh_logo_black.svg

**Pull request merged**